Cython/Build/Tests/TestCyCache.py

```diff
@@ -1,12 +1,10 @@
-import difflib
 import glob
 import gzip
 import os
 import tempfile
 
-import Cython.Build.Dependencies
-import Cython.Utils
-from Cython.TestUtils import CythonTest, write_file, fresh_cythonize, relative_lines_from_file
+from Cython.TestUtils import (CythonTest, write_file, fresh_cythonize,
+                              relative_lines_from_file)
```

Comment:
Hope no one (@scoder) has to `blame` me for the change? :]
Or should I say `git blame`? ;D